src/probabilities_estimators/histograms/rectangular_binning.jl

```diff
@@ -20,14 +21,13 @@ struct RectangularBinning{E} <: AbstractBinning
 end
 
 function probabilities(x::Vector_or_Dataset, binning::RectangularBinning)
```

Comment:
This function doesn't actually return probabilities per bin. It returns a vector of length `length(x)`. See the failing tests. For example:

```julia
julia> x = rand(100); binning = RectangularBinning(3);

julia> probabilities(x, binning)
100-element Probabilities{Float64}:
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 ⋮
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01
 0.01

```

Reply:
It works for `Dataset`s, though, so I think the problem is in how scalar-valued vectors inputs are handled. 

```julia
julia> x = Dataset([rand(3) for i = 1:100]); binning = RectangularBinning(2);

julia> probabilities(x, binning)
8-element Probabilities{Float64}:
 0.1
 0.13
 0.12
 0.08
 0.15
 0.13
 0.16
 0.13
```

Reply:
That's a bug we have to fic then!